hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -39,6 +39,7 @@
   // Validate contents of Metadata Table on each access against the actual filesystem
   public static final String METADATA_VALIDATE_PROP = METADATA_PREFIX + ".validate";
   public static final boolean DEFAULT_METADATA_VALIDATE = false;
+  public static final boolean DEFAULT_METADATA_ENABLE_FOR_READERS = false;
```

Comment:
can we move this closer to the actual property for which this is the default property?

Reply:
This is already next to `METADATA_VALIDATE_PROP` for which this is the default (for readers).